Core/src/AnonymousCredentials.php

```diff
@@ -64,4 +64,15 @@ public function getLastReceivedToken()
     {
         return $this->token;
     }
+
+    /**
+     * Returns the client name, in this case an empty string.
+     *
+     * @param callable $httpHandler
+     * @return string
+     */
+    public function getClientName(callable $httpHandler = null)
```

Comment:
What does this do?

Reply:
It fulfills the [`FetchAuthTokenInterface`](https://github.com/jdpedrie/google-auth-library-php/blob/signer/src/FetchAuthTokenInterface.php#L63) method used to retrieve a client email. [Credentials](https://github.com/jdpedrie/google-auth-library-php/tree/signer/src/Credentials) types which implement `SignBlobInterface` will support signing strings, and `getClientName()` will return the service account email.

Reply:
gotcha.